source/content/guides/drupal-9-migration/01-introduction.md

```diff
@@ -18,4 +18,53 @@ Drupal 9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://s
 
 There are a number of ways to upgrade to Drupal 9 on Pantheon, and we're building out the documentation within this guide to try to help you through your adventure.
 
+## Create a New Drupal 9 Site on Pantheon
+
 See our [Drupal 9](/drupal-9) and [Integrated Composer](/integrated-composer) docs to learn more about Drupal 9 on Pantheon, and to get started with a fresh new site.
+
+## Migrate or Upgrade an Existing Site to Drupal 9 With Integrated Composer on Pantheon
+
+<TabList>
+
+<Tab title="On Pantheon" id="on-pantheon" active={true}>
+
+Utilize the info on this tab if your site is currently on Pantheon.
+
+Here's how to migrate your site if you are using:
+
+- Build Tools (assumes Composer?)
+  - [Continuous Integration](/guides/drupal-9-migration/build-tools-to-d9-build-tools)
+  - No Continuous Integration
```

Comment:
@EdwardAngert - Was not able to find a relevant link- could you give a look?

Reply:
just confirmed - it's scheduled in DOC-357 as work that hasn't been started yet